source/system/io/File.ooc

```diff
@@ -299,9 +299,31 @@ File: abstract class {
 		Exception new(This, "Unsupported platform!\n") throw()
 		null
 	}
+	copy: static func (originalPath, newPath: String) {
+		originalFile := This new(originalPath)
+		newFile := This new(newPath)
+		originalFile copyTo(newFile)
+		(originalFile, newFile) free()
+	}
+	exists: static func ~file (path: String) -> Bool {
+		file := This new(path)
+		result := file exists()
+		file free()
+		result
+	}
+	rename: static func (original, new: String) {
+		This copy(original, new)
+		This rm(original)
```

Comment:
There is a `rename` function in `stdio` ( http://linux.die.net/man/3/rename ), maybe we could use it ?